### ISSUE TITLE
[Feature] Themes as a long-term UI design solution

### ISSUE THREAD

**Issue by SchaffRita** [Ayagikei/LifeUp#33]
**Is your feature request related to a problem? Please describe.**
Currently, the application is using material design principles; it could be a UI kit that is enhanced with some extra images, and developments.
I personally really like it, there are a couple of parts that should be adjusted to make it more coherent. As I checked the lab version of the new design (Material You - Material Design 3), I found that I don't really like the new design. 

A standard material design should not be discarded because it is appealing to a large audience. At the same time, there are those who may prefer other, quirkier game-like designs.

**Describe the solution you'd like**
A long-term solution could be making the application themeable. Meaning that the app could come with the basic (current) UI design and other themes would be available to turn on/off. This would give an extra edge over similar applications as well.

Still, if if it is feasible, it is a lot of work. So there comes my other idea to make this work: this is something that could give you more income. New skins/themes could be offered for a one-time fee (by theme). The ones who bought the app do not mind spending a little on this, and I personally would not mind paying a little extra for a UI I like.
It does not take away any useful features from the existing users either.

**Describe alternatives you've considered**
I can help you with the next design.

**Comment by Ayagikei** [comment]
Hello Rita,
Sorry for not replying for a long time.

In this regard, the diverse theme is indeed an interesting feature point.

But on a technical level, we currently don't know how to implement this UI switching logic.

Beyond that, we're still adapting Material 3 features. This could be a big project.

So this point may need to be discussed for a long time.

**Comment by SchaffRita** [comment]
Indeed, the current solution is really nice, I like how colors can be changed.
